doc/specs/#364 - Feature Toggle.md

```diff
@@ -43,6 +43,18 @@ Internally, a single flag enum will be used to reference experimental features.
 
 A feature is released by removing the feature enum value in code, and all related references. The value in settings will become ignored, and we can keep a list in a comment to prevent future reuse of settings names.
 
+## Settings File
+
+To enable experimental features a user will need to modify the settings file. The settings file can be opened via the settings command.
+
+```
+  "experimentalFeatures": {
```

Comment:
Have a look at https://github.com/PowerShell/PowerShell-RFC/blob/master/5-Final/RFC0029-Support-Experimental-Features.md to see how the PowerShell team has solved experimental feature handling.